pkgs/tools/security/tcplay/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGithub, cmake, pkgconfig, libgcrypt, libuuid, devicemapper, libudev, openssl, libgpgerror }:
+
+stdenv.mkDerivation rec {
+  name = "tcplay-2.0";
```

Comment:
To make updating easier, you could factor out a `version` attribute and use it here and in `rev` below.